### ISSUE TITLE
cabal-head prerelase date and commit are not updated

### ISSUE THREAD

**Issue by ulysses4ever** [haskell/cabal#10537]
Follow-up on https://github.com/haskell/cabal/issues/10515

The new release action doesn't update the date on the prerelease itself nor the commit it says was used to create the prerelease. It only updates the binaries and that's it...
![photo_2024-11-07_11-20-44](https://github.com/user-attachments/assets/7b5e6e35-64bd-4f9b-84f0-030273b97395)
(https://github.com/haskell/cabal/commit/0d80d5232fe6ff7a4eb4c6e5a42d643fad4ea98f is from October 3rd.)

We probably should explicitly remove the old prerelease before calling the release action (e.g. via https://github.com/marketplace/actions/delete-a-release).

**Comment by ulysses4ever** [comment]
after several attempts at fixing it, it's still not right for some reason...

**Comment by geekosaur** [comment]
https://github.com/marketplace/actions/nightly-release
https://github.com/marketplace/actions/github-release
https://github.com/actions/create-release
are some alternatives

**Comment by ulysses4ever** [comment]
Right, would be great to try it out. I don't think I'll have time for it any time soon though :-( /cc @TeofilC

**Comment by ulysses4ever** [comment]
It seems to work properly now, thanks @TeofilC! 

![Image](https://github.com/user-attachments/assets/0f311250-39b5-406e-a90c-79eae4f42ebf)

The "No release body provided" looks a bit unfriendly --- maybe we should put something nicer there. But that's far less annoying than the original issue.
